project-fortis-pipeline/docs/sp-setup.md

```diff
@@ -0,0 +1,28 @@
+# Azure Service Principal

```

Comment:
I found existing documentation for this https://github.com/Azure/acs-engine/blob/e3b22c480c0298b2cf1dce41c4a7e87ac49f46f3/docs/serviceprincipal.md

Would that document be good enough?

Reply:
yes I prefer this one and will link it. it's missing a note about what role you need `Owner` or `User Access Admin Role`, so ill just add that part.